### DISCUSSION TITLE
How to generate icons and splash screen?

### DISCUSSION THREAD

**Discussion by stevecassidy** [ionic-team/capacitor#4315] [Q&A]
I'm working on my first Capacitor based app which is wrapping a React web application (I build the web app first, then added Capacitor). All is working well but I'm finding the documentation hard to navigate with many references to Cordova which sometimes seem relevant but aren't clear how they apply. 

One question is about how to generate and install app icons and splash screen for my app.  I find a reference to `ionic cordova resources` in the ionic CLI documentation but I have no idea if that is relevant.  I'm not using the ionic CLI and when I tried to install it, it wants me to login to ionic and initialise the project in different ways etc.  So, not the right way to do it I think.

I see on here some reference in November last year to a new tool that helps with splash screen generation.  

Any pointers appreciated. 

Thanks,
Steve

**Comment by jcesarmobile** [comment]
This is the official documentation for splash and icons

https://capacitorjs.com/docs/guides/splash-screens-and-icons

It uses `cordova-res`, despite the name, it also supports Capacitor, the package was created before Capacitor existed, and I think there are plans to change its name.
